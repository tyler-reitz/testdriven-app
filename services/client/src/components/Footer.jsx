import React from 'react'

import './Footer.css'

const Footer = (props) => (
  <footer className="footer">
    <div className="container">
      <small className="has-text-grey">
        <span>Copyright 2019 TestDriven.io</span>
      </small>
    </div>
  </footer>
)

export default Footer
