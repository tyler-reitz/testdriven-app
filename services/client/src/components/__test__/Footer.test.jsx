import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

import Footer from '../Footer'

it('Footer renders properly', () => {
  const wrapper = shallow(<Footer />)
  expect(wrapper.length).toBe(1)
})

it('Show copyright info', () => {
  const wrapper = shallow(<Footer />)
  const element = wrapper.find('span')
  expect(element.length).toBe(1)
  expect(element.text()).toBe('Copyright 2019 TestDriven.io')
})

it('Footer renders a snapshot properly', () => {
  const tree = renderer.create(<Footer />).toJSON()
  expect(tree).toMatchSnapshot()
})
