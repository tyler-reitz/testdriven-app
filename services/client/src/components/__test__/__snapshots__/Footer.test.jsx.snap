// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders a snapshot properly 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <small
      className="has-text-grey"
    >
      <span>
        Copyright 2019 TestDriven.io
      </span>
    </small>
  </div>
</footer>
`;
